ACHNBrowserUI/ACHNBrowserUI/views/customTasks/CustomTasksListView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  CustomTasksListView.swift
+//  ACHNBrowserUI
+//
+//  Created by Jan on 24.05.20.
+//  Copyright © 2020 Thomas Ricouard. All rights reserved.
+//
+
+import SwiftUI
+import SwiftUIKit
+import Backend
+
+struct CustomTasksListView: View {
+    @EnvironmentObject private var collection: UserCollection
+    @Environment(\.presentationMode) private var presentationMode
+    @State private var editMode = EditMode.inactive
+    
+    private var closeButton: some View {
+        Button(action: {
+            self.presentationMode.wrappedValue.dismiss()
+        }, label: {
+            Image(systemName: "xmark.circle.fill")
+                .style(appStyle: .barButton)
+                .foregroundColor(.acText)
+        })
+        .buttonStyle(BorderedBarButtonStyle())
+        .accentColor(Color.acText.opacity(0.2))
+        .safeHoverEffectBarItem(position: .leading)
+    }
+    
+    private var orderButton: some View {
+        Button(action: {
+            if (self.editMode == EditMode.active) {
+                self.editMode = EditMode.inactive
+            }
+            else {
+                self.editMode = EditMode.active
+            }
+        }, label: {
+            if self.editMode == EditMode.active {
+                Image(systemName: "checkmark.seal.fill")
+                    .style(appStyle: .barButton)
+                    .foregroundColor(.acText)
+            }
+            else {
+                Image(systemName: "arrow.up.arrow.down.circle.fill")
+                    .style(appStyle: .barButton)
+                    .foregroundColor(.acText)
+            }
+        })
+        .buttonStyle(BorderedBarButtonStyle())
+        .accentColor(Color.acText.opacity(0.2))
+        .safeHoverEffectBarItem(position: .leading)
+    }
+    
+    var body: some View {
+        NavigationView {
+            List {
+                Group {
+                    ForEach(collection.dailyCustomTasks.tasks) { task in
+                        NavigationLink(destination: CustomTaskFormView(editingTask: task)) {
+                            CustomTaskRow(task: task)
+                        }
+                    }
+                    .onMove(perform: onMove)
+                    .onDelete { indexes in
+                        self.collection.deleteCustomTask(at: indexes.first!)
+                    }
```

Comment:
avoid force unwrapping (even we are 100% sure it's safe, I always prefer an extra guard rather than a force unwrap)
```suggestion
                    .onDelete { indexes in
                        guard let index = indexes.first else { return }
                        self.collection.deleteCustomTask(at: index)
                    }
```
However, in the application, we used force unwrapping (which I try to avoid), so you could ignore this comment. Up to you :)